docs/medical-api/more-info/webhooks.mdx

```diff
@@ -21,6 +21,17 @@ You should expect to get more than one Webhook message per patient per request (
 To enable this integration approach with Metriport, and for some prerequesite reading to understand
 how the Webhook flow works, see [our Webhooks guide](/home/api-info/webhooks).
 
+### Types of Messages
+
+- `medical.document-download`: result of Document Query, containing the newly downloaded documents
+   for the patient - see [details](#patient-document-data) below;
+- `medical.document-conversion`: result of converting the newly downloaded C-CDA documents into FHIR -
+   see [details](#patient-document-data) below;
+- `medical.document-bulk-download-urls`: list of download urls for a patient's documents, see 
+   [details](#bulk-document-download-urls) below;
+- `medical.consolidated-data`: result of a Consolidated Data Query, containing the patient's data in FHIR
+   format - see [details](#patient-consolidated-data) below.
+
```

Comment:
while we're at it - let's add mention of these types to each endpoint page? 

ie for example https://docs.metriport.com/medical-api/api-reference/document/start-document-query should mention that `medical.document-download` and `medical.document-conversion` will be sent, and we can link do this page for details